tests/components/sigfox/test_sensor.py

```diff
@@ -33,39 +30,32 @@
 """
 
 
-class TestSigfoxSensor(unittest.TestCase):
-    """Test the sigfox platform."""
-
-    def setUp(self):
-        """Initialize values for this testcase class."""
-        self.hass = get_test_home_assistant()
-        self.addCleanup(self.hass.stop)
+async def test_invalid_credentials(hass):
+    """Test for invalid credentials."""
+    with requests_mock.Mocker() as mock_req:
+        url = re.compile(API_URL + "devicetypes")
+        mock_req.get(url, text="{}", status_code=401)
+        assert await async_setup_component(hass, "sensor", VALID_CONFIG)
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6285789)